### DISCUSSION TITLE
How to force check a torrent

### DISCUSSION THREAD

**Discussion by NirKli** [Novik/ruTorrent#2506] [Q&A]
Hi,
I using crazyMax setup, and it seems there's something broken with rutorrent. 
I can't force check a torrent because its not on the downloading folder, it been moved to complete folder.
How can I force check a torrent?

ruTorrent version:4.1.5

**Comment by stickz** [comment]
This sounds like a bug with ruTorrent. I would recommend filing an issue report.

**Reply by NirKli** [reply]
@stickz 
Done.
[https://github.com/Novik/ruTorrent/issues/2507](url)
